packages/atomic-react/package.json

```
@@ -12,15 +12,14 @@
     "build": "nx build",
     "clean": "rimraf -rf dist",
     "build:bundles:esm": "tsc -p tsconfig.esm.json",
-    "build:bundles:cjs": "tsc -p tsconfig.cjs.json",
-    "build:bundles:iife": "rollup --config rollup.config.mjs",
-    "build:bundles": "concurrently \"npm run build:bundles:esm\" \"npm run build:bundles:cjs\"  \"npm run build:bundles:iife\"",
+    "build:bundles:iife-cjs": "rollup --config rollup.config.mjs",
```

IIFE is technically neither CJS nor ESM, as it is the antithesis of modules, where the code is "all gobbled up together".
We do indeed exposes it through browser>require, but that's more a pass of hand, which make me think:

Should we instead have an /iife subpath export?
`@coveo/packageName[/useCase]/iife` where `[/useCase]` is optional.

My rationale is that it'd be more descriptive I reckon 🤔

---

this means : "bundle for iife AND cjs using the rollup.config.mjs"

I am not sure what you mean, this part here is just changing how the bundling is done for CJS because of issues from "moduleResolution".

---

just talking about naming things :)
```suggestion
    "build:bundles:iife": "rollup --config rollup.config.mjs",
```

IMO